### PR TITLE
Add Github Actions build

### PR DESCRIPTION
**Pull request by ranisalt**

### CODE REVIEW

**Comment by hoangvvo** [comment]
I think `node-gyp` does not play well with MSVS 2019. I remember trying to build with 2019 on my laptop and needed to do some additional steps.

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
+        os: [ubuntu-latest, macos-latest, windows-latest]
```
Comment (hoangvvo) [inline .github/workflows/nodejs.yml]: Try `windows-2016` instead of `windows-latest`

Comment (ranisalt) [reply .github/workflows/nodejs.yml]: That indeed works. Argh, Windows for development sucks.

**Comment by hoangvvo** [comment]
Hey @ranisalt, after series of monkey-patching I finally get travis ci to do its job 🤣
https://github.com/hoangvvo/node-argon2/releases/tag/v0.25.0

I mess some thing up on the last PR.
Do you want me to make a PR to have Travis CI while you're figuring out Github Action build?

**Comment by ranisalt** [comment]
Well, I think there's no point in fixing Travis if we move away from it. I'll absorb your fixes and try to publish with GH Actions instead.

**Comment by hoangvvo** [comment]
@ranisalt I tried to look into GH Actions but the documentaion seems to be scarce... Can't really understand GH Actions syntax (and I'm already having trouble in Travis 🤣)

I was looking for a variable to determine if a build contains a tag but could not.

**Pull request closed without merging**